activestorage/lib/active_storage/previewer/pdf_previewer.rb

```diff
@@ -3,7 +3,22 @@
 module ActiveStorage
   class Previewer::PDFPreviewer < Previewer
     def self.accept?(blob)
-      blob.content_type == "application/pdf"
+      blob.content_type == "application/pdf" &&
+        mutool_exists?
+    end
+
+    def self.mutool_path
+      ActiveStorage.paths[:mutool] || "mutool"
+    end
+
+    # There's no version or way to get status code 0
+    # without processing a PDF. This check ensures
+    # the binary exists since command not found is
+    # exit code 127.
+    def self.mutool_exists?
```

Comment:
Let's memoize the return of this to avoid spawning a new process every time we want to generate a preview.

Reply:
Also we need to avoid the commands write to stdout/stderr.